rpcd/playbooks/verify-maas.yml

```diff
@@ -52,3 +52,6 @@
       delay: "5"
       when:
         - maas_use_api | bool
+
+  vars:
+    maas_use_api: true
```

Comment:
I'd rather leave it as False by default, and override it to True on some cases in our gating "contract" script.